lib/search_config.rb

```diff
@@ -10,6 +10,8 @@
 require "erb"
 
 class SearchConfig
+  ParseError = BaseParameterParser::ParseError
```

Comment:
Nope, we don't need it.

At the time I was just thinking it was weird for app.rb to have to know about the internals of parsing but this isn't a great solution. I think I'll just revert this.